code/modules/reagents/reagent_containers/food/condiment.dm

```diff
@@ -115,3 +115,135 @@
+
+/obj/item/reagent_containers/food/condiment/small/packet/ketchup
+	name = "ketchup packet"
+	desc = "Contains 5u of ketchup."
+	icon_state = "packet_small_red"
+	reagents_to_add = list(/decl/reagent/nutriment/ketchup = 5)
+
+/obj/item/reagent_containers/food/condiment/small/packet/mayo
+	name = "mayonnaise packet"
+	desc = "Contains 5u of mayonnaise."
+	icon_state = "packet_small_white"
+	reagents_to_add = list(/decl/reagent/nutriment/mayonnaise = 5)
+
+/obj/item/reagent_containers/food/condiment/small/packet/soy
+	name = "soy sauce packet"
+	desc = "Contains 5u of soy sauce."
+	icon_state = "packet_small_black"
+	reagents_to_add = list(/decl/reagent/nutriment/soysauce = 5)
+
+/obj/item/reagent_containers/food/condiment/small/packet/coffee
+	name = "instant coffee powder packet"
+	desc = "Contains 5u of instant coffee powder. Mix with 25u of water."
+	reagents_to_add = list(/decl/reagent/nutriment/coffeegrounds = 3)
+
+/obj/item/reagent_containers/food/condiment/small/packet/tea
+	name = "instant tea powder packet"
+	desc = "Contains 5u of instant black tea powder. Mix with 25u of water."
+	reagents_to_add = list(/decl/reagent/nutriment/teagrounds = 5)
+
+/obj/item/reagent_containers/food/condiment/small/packet/cocoa
+	name = "cocoa powder packet"
+	desc = "Contains 5u of cocoa powder. Mix with 25u of water and heat."
+	reagents_to_add = list(/decl/reagent/nutriment/coco = 5)
+
+/obj/item/reagent_containers/food/condiment/small/packet/grape
+	name = "grape juice powder packet"
+	desc = "Contains 5u of powdered grape juice. Mix with 15u of water."
+	reagents_to_add = list(/decl/reagent/nutriment/instantjuice/grape = 5)
+
+/obj/item/reagent_containers/food/condiment/small/packet/orange
+	name = "orange juice powder packet"
+	desc = "Contains 5u of powdered orange juice. Mix with 15u of water."
+	reagents_to_add = list(/decl/reagent/nutriment/instantjuice/orange = 5)
+
+/obj/item/reagent_containers/food/condiment/small/packet/watermelon
+	name = "watermelon juice powder packet"
+	desc = "Contains 5u of powdered watermelon juice. Mix with 15u of water."
+	reagents_to_add = list(/decl/reagent/nutriment/instantjuice/watermelon = 5)
+
+/obj/item/reagent_containers/food/condiment/small/packet/apple
+	name = "apple juice powder packet"
+	desc = "Contains 5u of powdered apple juice. Mix with 15u of water."
+	reagents_to_add = list(/decl/reagent/nutriment/instantjuice/apple = 5)
+
+/obj/item/reagent_containers/food/condiment/small/packet/protein
+	name = "protein powder packet"
+	desc = "Contains 10u of powdered protein. Mix with 20u of water."
+	icon_state = "packet_medium"
+	reagents_to_add = list(/decl/reagent/nutriment/protein = 10)
+
+/obj/item/reagent_containers/food/condiment/small/packet/crayon
+	name = "crayon powder packet"
+	desc = "Contains 10u of powdered crayon. Mix with 30u of water."
+	reagents_to_add = list(/decl/reagent/crayon_dust = 10)
+
+/obj/item/reagent_containers/food/condiment/small/packet/crayon/red
```

Comment:
please add a newline between each crayon colour entry to improve code readability.